xml/MAIN-SBP-Migrate-z-KVM.xml

```diff
@@ -61,11 +61,11 @@
         <para>Like KVM for IBM z Systems, KVM integrated with SUSE Linux Enterprise Server for z Systems and LinuxONE is
             supported when installed into an LPAR. The section <quote>Installation on IBM z
                 Systems</quote> of the SUSE Linux Enterprise Server Deployment Guide at <link
-                xlink:href="https://www.suse.com/documentation/sles-12/book_sle_deployment/data/cha_zseries.html"
+                xlink:href="https://documentation.suse.com/sles/12-SP4/html/SLES-all/cha-zseries.html"
```

Comment:
This guide is for 12 SP2, but the link is for 12 SP4. @chabow, can you suggest which of the following to use here?:

* redirect to newest SLES 12 SP (12 SP5): https://documentation.suse.com/sles-12/...

* direct link to SLES 12 SP2: https://documentation.suse.com/sles/12-SP2/...

Reply:
Hi  - this paper MAIN-SBP-Migrate-z-KVM.xml is not online/published anymore, because it is outdated. I will not remove it from the config file as I don't know yet if it is worth to be updated. Thus it is outcommented only. But it will not be published as is, thus I didn't update the links originally.

Reply:
And an update of the links is not necessary.